packages/core/framework/src/container.ts

```diff
@@ -1,18 +1,13 @@
 import { createMedusaContainer } from "@medusajs/utils"
 import { AwilixContainer, ResolveOptions } from "awilix"
-import { TransformObjectMethodToAsync } from "@medusajs/types";
-
-/**
- * The following interface acts as a bucket that other modules or the
- * utils package can fill using declaration merging
- */
-export interface ModuleImplementations {}
+import { ModuleImplementations } from "@medusajs/types";
 
 /**
  * The Medusa Container extends [Awilix](https://github.com/jeffijoe/awilix) to
  * provide dependency injection functionalities.
  */
-export type MedusaContainer<Cradle extends object = TransformObjectMethodToAsync<ModuleImplementations>> =
+// export type MedusaContainer<Cradle extends object = TransformObjectMethodToAsync<ModuleImplementations>> =
+export type MedusaContainer<Cradle extends object = ModuleImplementations> =
```

Comment:
@adrien2p I have commented this out for now. Let's have a chat tomorrow and see how we can workaround this. I will not merge until then